.github/workflows/build-publish-tag-final-image.yaml

```
@@ -7,7 +7,7 @@ on:
         description: "Final Image Tag to Deploy to docker hub"
         default: "1.1.1"
         type: string
-
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)